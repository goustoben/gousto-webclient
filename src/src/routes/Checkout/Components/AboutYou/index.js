export default from './AboutYouContainer'
export * from './AboutYouContainer'
