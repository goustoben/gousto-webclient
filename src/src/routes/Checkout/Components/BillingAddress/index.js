export { BillingAddressContainer as BillingAddress } from './BillingAddressContainer'
