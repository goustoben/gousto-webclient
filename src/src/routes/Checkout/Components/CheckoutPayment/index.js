export { CheckoutPaymentContainer as CheckoutPayment } from './CheckoutPaymentContainer'
