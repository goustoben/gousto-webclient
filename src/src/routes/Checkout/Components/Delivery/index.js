export { DeliveryContainer, validationMessages, addInitialValues } from './DeliveryContainer'
