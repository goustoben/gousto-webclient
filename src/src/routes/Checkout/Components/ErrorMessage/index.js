export { ErrorMessageContainer as ErrorMessage } from './ErrorMessageContainer'
