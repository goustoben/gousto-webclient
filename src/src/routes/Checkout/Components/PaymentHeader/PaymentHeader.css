@value ScreenXXSMin from 'styles/screenSizes.css';

.headerContainer {
	display:flex;
	flex-wrap: wrap;
	justify-content: space-between;
}

.header {
	composes: headerMD headlineFont from 'styles/typography.css';
	composes: inlineBlock from 'styles/display.css';
}

.padlockIcon {
	composes: padlockIcon from 'styles/icons.css';
}

.textSM {
	composes: textSM from 'styles/typography.css';
	composes: noMargin from 'sass-loader!styles/spacing.scss';
}

.boldInfo{
	composes: textSM from 'styles/typography.css';
	composes: marginXSBottom from 'sass-loader!styles/spacing.scss';
}

