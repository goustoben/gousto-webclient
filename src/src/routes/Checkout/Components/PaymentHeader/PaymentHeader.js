import React from 'react'

import css from './PaymentHeader.css'

export const PaymentHeader = () => (
  <div>
    <div className={css.headerContainer}>
      <h3 className={css.header}>Payment details <span className={css.padlockIcon} /></h3>
      <span className={css.boldInfo}>All fields are required</span>
    </div>
    <p className={css.textSM}>All transactions are secured using 128 bit SSL technology.</p>
  </div>
)
