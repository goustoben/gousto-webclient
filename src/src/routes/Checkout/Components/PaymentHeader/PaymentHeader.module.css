@value ScreenXXSMin from 'styles/screenSizes.module.css';

.paymentHeader {
  margin-bottom: 25px;
}

.headerContainer {
  display: flex;
  flex-wrap: wrap;
  justify-content: space-between;
}

.header {
  composes: headerMD headlineFont from 'styles/typography.module.css';
  composes: inlineBlock from 'styles/display.module.css';
}

.padlockIcon {
  composes: padlockIcon from 'styles/icons.module.css';
}

.textSM {
  composes: textSM from 'styles/typography.module.css';
  composes: noMargin from 'sass-loader!styles/spacing.scss';
}

.boldInfo {
  composes: textSM from 'styles/typography.module.css';
  composes: marginXSBottom from 'sass-loader!styles/spacing.scss';
}
