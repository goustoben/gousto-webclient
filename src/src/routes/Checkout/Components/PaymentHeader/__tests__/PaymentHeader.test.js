import React from 'react'
import renderer from 'react-test-renderer'

import { PaymentHeader } from 'routes/Checkout/Components/PaymentHeader'

describe('PaymentHeader', () => {
  test('will match snapshot', () => {
    const tree = renderer.create(<PaymentHeader />).toJSON()

    expect(tree).toMatchSnapshot()
  })
})
