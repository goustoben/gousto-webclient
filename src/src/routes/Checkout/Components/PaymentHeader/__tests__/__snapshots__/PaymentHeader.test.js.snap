// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PaymentHeader will match snapshot 1`] = `
<div>
  <div
    className="headerContainer"
  >
    <h3
      className="header"
    >
      Payment details
       
      <span
        className="padlockIcon"
      />
    </h3>
    <span
      className="boldInfo"
    >
      All fields are required
    </span>
  </div>
  <p
    className="textSM"
  >
    All transactions are secured using 128 bit SSL technology.
  </p>
</div>
`;
